pocs/observatory.py

```diff
@@ -39,10 +37,8 @@ def __init__(self, cameras=None, scheduler=None, dome=None, *args, **kwargs):
         self.observer = None
         self._setup_location()
 
-        self.logger.info('\tSetting up mount')
-        self.mount = None
-        self._create_mount()
-
+        self.logger.info('\tAdding mount to Observatory.')
+        self.mount = mount
```

Comment:
Use the method that you created, which will also generate the appropriate log message.

```suggestion
        self.set_mount(mount)
```